deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -2021,6 +2021,15 @@ extension ChatViewController {
         menuElements.append(action)
     }
 
+    private func isLinkTapped(indexPath: IndexPath, point: CGPoint) -> String? {
+        if let cell = tableView.cellForRow(at: indexPath) as? BaseMessageCell {
```

Comment:
Why `if` instead of `guard`?

Reply:
that comes from switching 5+ languages all the time :)